tools/markdown-to-html/BUILD.bazel

```diff
@@ -1,24 +1,26 @@
 load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
-load("//tools:defaults.bzl", "jasmine_node_test", "ts_library")
+load("//tools:defaults.bzl", "jasmine_node_test")
+load("//tools:defaults2.bzl", "ts_project")
 
 package(default_visibility = ["//visibility:public"])
 
-ts_library(
+ts_project(
     name = "transform-markdown",
     srcs = glob(
         ["**/*.ts"],
         exclude = [
             "*.spec.ts",
         ],
     ),
-    # TODO(ESM): remove this once the Bazel NodeJS rules can handle ESM with `nodejs_binary`.
-    devmode_module = "commonjs",
-    tsconfig = ":tsconfig.json",
+    args = [
```

Comment:
Why delete the config for `markdown-to-html` and use these args instead of just having the extending the tsconfig

Reply:
Ops. good call. Deleted that as we don't need it.